tests/integ/test_function.py

```diff
@@ -1742,6 +1743,10 @@ def test_array_negative(session):
         df.select(array_size([1])).collect()
     assert "'ARRAY_SIZE' expected Column or str, got: <class 'list'>" in str(ex_info)
 
+    with pytest.raises(TypeError) as ex_info:
+        df.select(size([1])).collect()
+    assert "'SIZE' expected Column or str, got: <class 'list'>" in str(ex_info)
+
     with pytest.raises(TypeError) as ex_info:
         df.select(array_slice([1], "col1", "col2")).collect()
```

Comment:
nit: Where are the tests for the new function and all of its code paths?

Reply:
they are added as doctests